### PR TITLE
moved existing documentation files into a new subfolder 

### PR DESCRIPTION
**Pull request by jenmalloy**
and added new files for new docs content and docs build

### CODE REVIEW

**Comment by lulf** [comment]
The stuff in the logo folder is referenced from the top level README, so the link should be updated (also on the github pages repo) . Would it make sense to move the logo folder to the root ?

File: documentation/design_docs/README.adoc [hunk 1]
```diff
@@ -3,29 +3,29 @@ EnMasse
 -------
 
 Documents relating to EnMasse project. For setting up EnMasse, see
-link:getting-started/README.adoc[Getting Started]. To setup EnMasse with
+link:design_docs/getting-started/README.adoc[Getting Started]. To setup EnMasse with
```
Comment (grs) [inline documentation/design_docs/README.adoc]: I don't think these links should have the design_docs prefix, as the doc they are in is itself in that design_docs folder. It is the toplevel README, i.e. directly under enmasse, that needs its links updated.

Comment (jenmalloy) [reply documentation/design_docs/README.adoc]: @grs, got it, thanks, Gordon! Will revert this one and fix the top-level README file.

**Comment by lulf** [comment]
Closing as I believe this has been merged in a different PR

**Pull request closed without merging**